homeassistant/components/smartthings/media_player.py

```diff
@@ -0,0 +1,228 @@
+"""Support for media players through the SmartThings cloud API."""
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import Any
+
+from pysmartthings import Capability, DeviceEntity
+
+from homeassistant.components.media_player import (
+    DOMAIN as MEDIA_PLAYER_DOMAIN,
+    MediaPlayerEntity,
+    MediaPlayerEntityFeature,
+    MediaPlayerState,
+    RepeatMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SmartThingsEntity
```

Comment:
```suggestion
from .entity import SmartThingsEntity
```

Reply:
@alrassia Thanks ! But please wait, I'm updating the code right now. I had to make a PR in order to rebase :)